### DISCUSSION TITLE
MASS of parts

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/MIR#60] [Q&A] [answered]
@Toa-Sampa 

have already adjusted the MIR core part 
it originally was mass = 17
  RO had set it to 15.88
  so scaled it using normal reverse RO math for mass and got a mass = 8.447
this is a reduction from current to suggested?? of 0.4968823529%

just found the original SovietPack by @BobCat, which has the MIR-core with a mass of 5. I believe we should go with those numbers (from the original pack) as much as possible. Seems when DECQ et al published their version (with permission) they made it much heavier on purpose (s.i.c)

Comments?

What about costs? same adjustment?
[SovietPackv1.9.1b-just-MIR.zip](https://github.com/zer0Kerbal/MIR/files/11670195/SovietPackv1.9.1b-just-MIR.zip)

mass = 6.93 // was 14 - following core: 0.4968823529%

**Comment by Toa-Sampa** [comment]
Van you give me the current costs of components?  That way we can see if
costs should be reduced looking at real cost

**Reply by zer0Kerbal** [reply, marked as answer]
all listed in the part.cfg's - from the original BobCat version to the DECQ version - seems cost hasn't changed (at least in the core part)
